src/main/scala/viper/silver/frontend/SilFrontEndConfig.scala

```diff
@@ -174,6 +181,10 @@ abstract class SilFrontendConfig(args: Seq[String], private var projectName: Str
     }
   }
 
+  // Since the smoke detection plugin depends on the refute plugin, smoke detection cannot be enabled when default
+  // plugins (which include the refute plugin) are disabled.
+  conflicts(enableSmokeDetection, List(disableDefaultPlugins))
```

Comment:
What if I disable the default plugins but manually re-enable the refute plugin?

Reply:
Yes, I also realized today that this is too restrictive. -> 98dace16